.circleci/config.yml

```diff
@@ -148,6 +187,7 @@ workflows:
   version: 2
   build-test-deploy:
     jobs:
-      - build-and-test    # Mac OS/X raw ("build-and-test")
+      - build-debug    # Mac OS/X raw ("build-debug")
```

Comment:
build-and-test was set as a Required target in GH, I'll change it to `build-debug` and `build-release`